src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,101 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
```

Comment:
checklist p.8 [STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size

Reply:
@liliia-livandovskaya

1. I already set paddings. Do I need to change "height" to "min-height", so in case if I get additional elements, card could get higher? Did I get it correct?

2. Can I leave fixed width, or I need to figure out another way to keep my card size? When I remove width, my card grow to full body width.